dispatch/apps/api/views.py

```diff
@@ -431,23 +431,32 @@ def get_queryset(self):
         return queryset
 
 
-@api_view(['POST'])
 @permission_classes((AllowAny,))
-def user_authenticate(request):
+class AuthenticationViewSet(viewsets.GenericViewSet):
```

Comment:
I like the idea of making it more consistent with the other REST resources.

I don't know django-rest-framework though so I imagine it'd take me a day or so to learn it well enough to make any changes.